### DISCUSSION TITLE
Instana Server Config not being picked instana-agent.customresource.yaml

### DISCUSSION THREAD

**Discussion by awasthan** [ppc64le/build-scripts#918] [General]
@amitsadaphule It seems like the Instana Server details (endpoint, port and agent-key) are not being picked up from the 'instana-agent.customresource.yaml'. Instead they are picked from 'com.instana.agent.main.sender.Backend.cfg' in the agent tarball. So it needs to build separate agent docker image for each env (as Instana servers are different for each environment). 
Can you please confirm this?

**Comment by amitsadaphule** [comment]
@awasthan looks like you're right. I tried removing the key from the `com.instana.agent.main.sender.Backend.cfg` and then creating the agent image. Post this change, my cluster no longer showed up on the dashboard. I tried adding the agent key, endpoint, port in the CR yaml as ENVs too as described [here](https://www.instana.com/docs/setup_and_manage/host_agent/on/kubernetes/#install-operator-manually) and [here](https://www.instana.com/docs/setup_and_manage/host_agent/on/docker/). But that didn't help either. I'd raise this issue on the instana agent operator github. Thanks!
